### ISSUE TITLE
should we use class: Todo instead of type: Todo in param declaration

### ISSUE THREAD

**Issue by sollycatprint** [ruby-hyperloop/hyper-react#131]
_From @catmando on February 29, 2016 20:54_

In writing documentation I realize we are at odds with Ruby terminology here.

Should we allow "class:" as an alias for "type:" in Param declarations?

If so should we eventually deprecate type?

_Copied from original issue: zetachang/react.rb#131_

**Comment by catmando** [comment]
@zetachang - I think we should at least offer class: as alternative to type:

**Comment by zetachang** [comment]
Agreed on this.  But the original inspiration is from https://github.com/ruby-grape/grape#supported-parameter-types , have no idea why they choose "type" instead of "class".
